examples/trace/src/main/java/com/google/cloud/opentelemetry/example/trace/TraceExample.java

```diff
@@ -0,0 +1,64 @@
+package com.google.cloud.opentelemetry.example.trace;
+
+import com.google.cloud.opentelemetry.trace.TraceConfiguration;
+import com.google.cloud.opentelemetry.trace.TraceExporter;
+import io.opentelemetry.OpenTelemetry;
+import io.opentelemetry.sdk.OpenTelemetrySdk;
+import io.opentelemetry.sdk.trace.export.SimpleSpanProcessor;
+import io.opentelemetry.trace.Span;
+import io.opentelemetry.trace.Tracer;
+
+import java.io.IOException;
+import java.time.Duration;
+
+public class TraceExample {
```

Comment:
TraceExample sounds a bit too general for me, how about TraceExporterExample?

Reply:
Good suggestion